spec/mail/mailbox.mdown

```diff
@@ -221,7 +232,7 @@ If the mailbox has `mayDelete == false`, the destruction MUST be rejected with a
 
 A mailbox MUST NOT be destroyed if it still has any child mailboxes. Attempts to do so MUST be rejected with a `mailboxHasChild` error.
 
-A mailbox MUST NOT be destroyed if it has any messages assigned to it. Attempts to do so MUST be rejected with a `mailboxHasMessage` error.
+Before destroying the mailbox, all messages currently assigned to it must be removed from the mailbox. If a message is not in any other mailboxes, it MUST be added to the mailbox given by the *onLastReferenceMoveTo* argument, if given, otherwise destroyed.
```

Comment:
Shouldn't "given" be "non-null"?

If not supplying the argument triggers implicit destruction of other messages, then the default is against JMAP's principles. Having that destruction happen if the client supplies an explicit null seems more acceptable.